### PR TITLE
Added instructions for registering model and support additional param overrides

### PR DESCRIPTION
**Pull request by VipulMascarenhas**
### Description

This PR updates the following documentation for AI Quick Actions:
- Add advanced fine-tuning options to support additional hyperparameters.
- model deployment instructions to accommodate details for TGI alongside vLLM inference/serving engine.
- Instructions to upload a model to object storage for model registration.
- Links to navigate to registration page.

### CODE REVIEW

**Review by mrDzurb — DISMISSED** [review]

File: ai-quick-actions/register-tips.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- [CLI](cli-tips.md)
+- [Policies](policies/README.md)
+- [Model Deployment](model-deployment-tips.md)
+- [Model Evaluation](evaluation-tips.md)
+- [Model Fine Tuning](fine-tuning-tips.md)
+
+## Upload model artifact to Object Storage
+
+AI Quick Actions (AQUA) supports user-provided models that can be deployed, fined-tuned and evaluated. You can now upload
+and test the models with the artifacts downloaded either from Hugging Face or from your own models created from scratch.
```
Comment (tsikiksr) [inline ai-quick-actions/register-tips.md:15]: I'd rephrase:
"You can now upload and test models with artifacts downloaded from model repositories like Hugging Face, etc. or from your own models"

File: ai-quick-actions/register-tips.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+While registering the model in AQUA, you need to specify the Object Storage location where the model artifact is stored.
```
Comment (tsikiksr) [inline ai-quick-actions/register-tips.md:17]: Don't use AQUA. only use "AI Quick Actions"

File: ai-quick-actions/register-tips.md (hunk 1 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/register-tips.md]: updated.

File: ai-quick-actions/register-tips.md (hunk 2 above)
Comment (VipulMascarenhas) [reply ai-quick-actions/register-tips.md]: updated.

**Comment by VipulMascarenhas** [comment]
@mrDzurb @darenr need approval again to merge this.

**Review by darenr — APPROVED** [review]
